apps/cookbook/src/app/examples/button-example/examples/_responsive-grid-layout.scss

```
@@ -0,0 +1,11 @@
+@use '@kirbydesign/core/src/scss/utils';
+
+@use './grid-layout';
+
+:host {
+  @include utils.media('<medium') {
+    display: inline-flex;
```

Why is the outer display type `inline`? It doesn't look like the layout is used with other inlined elements.

```suggestion
    display: flex;
```

---

As to not spread the buttons thin 😅 
I.e. I would like them to stack up from the left, with the gap in between.
`display: grid` would spread them horizontally across the available width.

---

This one uses flexbox. But as long as no other adjacent elements are displayed inline there's no issue with it 😊